src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacNumericValidator.java

```diff
@@ -228,8 +231,12 @@ private QuantityValidationResponse validateWithUnits(final IsaacNumericQuestion
                                                          final Quantity answerFromUser) {
         log.debug("\t[validateWithUnits]");
         QuantityValidationResponse bestResponse = null;
-        int sigFigsToValidateWith = numberOfSignificantFiguresToValidateWith(answerFromUser.getValue(),
-                isaacNumericQuestion.getSignificantFiguresMin(), isaacNumericQuestion.getSignificantFiguresMax());
+        Integer sigFigsToValidateWith = null;
+
+        if (!isaacNumericQuestion.getDisregardSignificantFigures()){
```

Comment:
CheckStyle will warn you about this now, but we prefer `) {` with a space!

Reply:
There are a few other places, but I won't spam this comment on them all.

Reply:
Updated with CheckStyle recommendations.